pages/_app.js

```diff
@@ -1,10 +1,49 @@
 import GlobalStyle from "../styles";
+import StyledToastContainer from "@/components/Styles/StyledToastContainer";
+import { useState } from "react";
+import { nanoid } from "nanoid";
+import Toast from "@/components/Toast";
 
 export default function App({ Component, pageProps }) {
+  const [toasts, setToasts] = useState([]);
+
+  function handleAddToast(message, variant = "success") {
+    const id = nanoid();
+    setToasts((prevToasts) => [
+      ...prevToasts,
+      { id, visible: true, message, variant },
+    ]);
+
+    setTimeout(() => {
+      setToasts((prevToasts) =>
+        prevToasts.map((toast) =>
+          toast.id === id ? { ...toast, visible: false } : toast
+        )
+      );
+    }, 4500);
+
+    setTimeout(() => {
+      handleDeleteToast(id);
+    }, 5000);
+  }
+
```

Comment:
Should the message disappear after 2/3 seconds?

Reply:
You are right. the timer is a little higher at the moment. We though this is a better timer than only 2/3 seconds :)